### ISSUE TITLE
configure_tensorflow.sh broken

### ISSUE THREAD

**Issue by kingvash** [tensorflow/minigo#501]
"The C++ Minigo port uses version 0.11.1 of the Bazel build system"

```
Configuring tensorflow
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.google.protobuf.UnsafeUtil (file:/home/eights/.cache/bazel/_bazel_eights/install/6338e49942acfd59d75bbd9bec01f065/_embedded_binaries/A-server.jar) to field java.nio.Buffer.address
WARNING: Please consider reporting this to the maintainers of com.google.protobuf.UnsafeUtil
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
You have bazel 0.11.1 installed.
Please upgrade your bazel installation to version 0.15.0 or higher to build TensorFlow!
Configuration finished
Building tensorflow package
.................
WARNING: Config values are not defined in any .rc file: opt
ERROR: /tmp/minigo_tf/third_party/gpus/cuda_configure.bzl:119:1: file '@bazel_tools//tools/cpp:windows_cc_configure.bzl' does not contain symbol 'setup_vc_env_vars'
ERROR: error loading package '': Extension file 'third_party/gpus/cuda_configure.bzl' has errors
ERROR: error loading package '': Extension file 'third_party/gpus/cuda_configure.bzl' has errors
INFO: Elapsed time: 1.694s
FAILED: Build did NOT complete successfully (0 packages loaded)

```

**Comment by kingvash** [comment]
upgraded to 0.17.2

`NameError: global name 'cuda_toolkit_path_full' is not defined`

**Comment by amj** [comment]
ok, we should update our readme :)  We've gotten it working with 0.17.2 and TF 1.11 recently.

re: the cuda path, that might be a bug on tensorflow proper.

**Comment by sethtroisi** [comment]
at HEAD I'm getting errors related to missing tensorrt files, this doesn't seem to be documented in the README
`bazel test --define=board_size=9 cc/...  &&  bazel test cc/...`

```
FAIL: //cc/dual_net:dual_net_test (see /home/eights/.cache/bazel/_bazel_eights/7a6714f43f855c94d0ec539c2bb695f8/execroot/__main__/bazel-out/k8-fastbuild/testlogs/cc/dual_net/dual_net_test/test.log)
ERROR: missing input file '@local_config_tensorrt//:tensorrt/include/NvUffParser.h'
ERROR: /home/eights/.cache/bazel/_bazel_eights/7a6714f43f855c94d0ec539c2bb695f8/external/local_config_tensorrt/BUILD:16:1: @local_config_tensorrt//:nvparsers: missing input file '@local_config_tensorrt//:tensorrt/include/NvUffParser.h'
ERROR: /home/eights/.cache/bazel/_bazel_eights/7a6714f43f855c94d0ec539c2bb695f8/external/local_config_tensorrt/BUILD:16:1 1 input file(s) do not exist
```

**Comment by tommadams** [comment]
I noticed that this morning and fixed it in #510.

cc/dual_net:trt_dual_net needed a `manual` tag to avoid it being compiled when you run `bazel build cc/...`
